src/uwtools/config.py

```diff
@@ -493,7 +492,21 @@ def _format_output(self):
         return "\n".join(outstring)
 
     def dump_file(self, output_path):
-        ''' Write the formatted output to a text file. '''
+        ''' Write the formatted output to a text file. 
+FMS field and tracer managers must be registered in an ASCII table called 'field_table'
+This table lists field type, target model and methods the querying model will ask for.
+
+See UFS documentation for more information:
+https://ufs-weather-model.readthedocs.io/en/ufs-v1.0.0/InputsOutputs.html#field-table-file
+
+The example format for generating a field file is::
+
+sphum:
+  longname: specific humidity
+  units: kg/kg
+  profile_type: 
+    name: fixed
+    surface_value: 1.e30'''
```

Comment:
Please do push this whole block over to be inline with the beginning of the doc block in column 8.

Reply:
Moved over 8 spaces

Reply:
I meant the entire section here. We want the first 8 columns of the whole doc block clear.